libfwupdplugin/fu-firmware.h

```diff
@@ -55,17 +55,57 @@ typedef enum {
 	FU_FIRMWARE_EXPORT_FLAG_UNKNOWN = G_MAXUINT64,
 } FuFirmwareExportFlags;
 
+/**
+ * FuFirmwareParseFlags:
+ *
+ * The firmware parse flags.
+ **/
+typedef enum {
+	/**
+	 * FWUPD_INSTALL_FLAG_NONE:
```

Comment:
I think the comment should have been updated to FU_FIRMWARE_PARSE_FLAG_NONE too.